src/main/java/codeu/model/store/persistence/PersistentStorageAgent.java

```diff
@@ -103,4 +103,5 @@ public void writeThrough(Conversation conversation) {
   public void writeThrough(Message message) {
     persistentDataStore.writeThrough(message);
   }
+
```

Comment:
Why is `PersistentStorageAgent` getting modified in this PR?

Reply:
`PersistentStorageAgent` got modified when `ActivityStore` existed- after its removal some spacing mess had been left over. It has been reverted back to its original.